lib/core/repository/categories/cms.rb

```diff
@@ -6,10 +6,15 @@ def initialize(options = {})
       end
 
       def all
-        response = connection.get('/api/%{locale}/categories.json' % { locale: I18n.locale })
+        categories_url = '/api/%{locale}/categories.json' % { locale: I18n.locale }
+        response = connection.get(categories_url)
         response.body
-      rescue
-        raise RequestError, 'Unable to fetch Category JSON from Contento'
+      rescue => e
+        raise RequestError,
+              'Unable to fetch Category JSON from Contento' +
+              " url: [#{categories_url}]" +
+              " error: [#{e.inspect}]" +
+              " url_prefix: [#{connection.try(:url_prefix).inspect}]"
```

Comment:
The connection, or ultimately `ENV['MAS_CMS_URL']. I was logging out the whole connection but it was a bit noisy.

This is what you get with this (in dev mode, obviously).

![voila_capture 2015-11-11_10-50-40_am](https://cloud.githubusercontent.com/assets/2986/11089543/7f4805b6-8862-11e5-9a00-57949faad4f0.png)